x-pack/plugins/security_solution/scripts/run_cypress/print_run.ts

```diff
@@ -356,6 +356,9 @@ export function renderSummaryTable(results: CypressCommandLine.CypressRunResult[
     });
 
     _.each(runs, (run) => {
+      // Handle ERROR TypeError: Cannot destructure property 'spec' of 'run' as it is undefined.
```

Comment:
out of curiosity: how do we not have `run` when we map over runs? This seems strange

Reply:
because the`runs` array can have `undefined` as an item